lib/register/transform.js

```diff
@@ -23,9 +23,9 @@ var transformTypes = ['name', 'value', 'attribute'];
  * @function
  * @param {Object} transform - Transform object
  * @param {String} transform.type - Type of transform, can be: name, attribute, or value
- * @param {String} transform.name - Name of the transformer so a transformGroup can call a list of transforms.
+ * @param {String} transform.name - Name of the transformer by which a transformGroup can call a list of transforms.
  * @param {Function} [transform.matcher] - Matcher function, return boolean if transform should be applied. If you omit the matcher function, it will match all properties.
- * @param {Function} transform.transformer Performs a transform on a property object, should return a string or object depending on the type. Will only update certain properties so you can't mess up property objects on accident.
+ * @param {Function} transform.transformer Performs a transform on a property object, should return a string or object depending on the type. Will only update certain properties by which you can't mess up property objects on accident.
```

Comment:
You can't mess up property objects on accident as it will only update certain properties.